libs/sdk-ui-pivot/src/impl/structure/colDefFactory.ts

```diff
@@ -231,6 +232,7 @@ function createColumnHeadersFromDescriptors(
                     headerName,
                     headerTooltip: headerName,
                     ...sortProp,
+                    sortingOrder: sortProp.sort === "asc" ? ["asc", "desc"] : ["desc", "asc"],
```

Comment:
These arrays are repeated. Can you make constants out of them and reuse them in both instances?

Reply:
fixed